src/index.js

```diff
@@ -57,13 +55,9 @@ export default function wdm(compiler, opts = defaults) {
 
   // start watching
   if (!options.lazy) {
-    context.watching = compiler.watch(options.watchOptions, (err) => {
-      if (err) {
-        context.log.error(err.stack || err);
-
-        if (err.details) {
-          context.log.error(err.details);
-        }
+    context.watching = compiler.watch(options.watchOptions, (error) => {
+      if (error) {
+        context.logger.error(error);
```

Comment:
Simplify error output. Also it is improve error message because error can have own `inspect` Symbol.